lighthouse-cli/bin.ts

```diff
@@ -276,11 +276,11 @@ function saveResults(results: Results,
     return promise.then(_ => Printer.write(results, flags.output, flags.outputPath));
 }
 
-function runLighthouse(url: string,
+export function runLighthouse(url: string,
                        flags: {port: number, skipAutolaunch: boolean, selectChrome: boolean, output: any,
                          outputPath: string, interactive: boolean, saveArtifacts: boolean, saveAssets: boolean
                          maxWaitForLoad: number},
-                       config: Object): Promise<undefined> {
+                       config: Object | null): Promise<undefined> {
 
   let chromeLauncher: ChromeLauncher;
```

Comment:
why is this declaration allowed by TS when we know it starts off as undefined :/